### ISSUE TITLE
Doesn't load on startup sometimes

### ISSUE THREAD

**Issue by DuBistKomisch** [joshuacant/tst-wheel_and_double#22]
Sorry for the vague issue, but I'm not really sure how to debug firefox extensions.

The extension works fine on another computer, so my guess is that it's because I keep 2 windows around (work and non-work) on this laptop. Toggling the extension off and back on fixes it.

**Comment by joshuacant** [comment]
This comes and goes. 

Not sure if it's changes to Firefox or TST or something I've done wrong. I've tried fixing it and never been able to 100% eliminate it. It was fine for over a year but recently I've been seeing it every once in a while with FF Dev Edition betas and the latest release version of TST.

I'll leave this issue open, maybe someone smarter than me can figure out a once-and-for-all fix.

**Comment by DuBistKomisch** [comment]
Yeah I'm using dev edition too. Thanks, happy to provide any info anyone needs.

**Comment by DuBistKomisch** [comment]
alright I've figured out how to inspect addons now, and it finally happened again:

![image](https://user-images.githubusercontent.com/703055/72851442-bf2dd380-3cff-11ea-96de-2318e13c9a36.png)

vs normal:

![image](https://user-images.githubusercontent.com/703055/72851492-e4badd00-3cff-11ea-86f8-112d84e2a65b.png)

**Comment by joshuacant** [comment]
Thanks for the logs. This problem has been getting worse for me again. I'm having to manually toggle it several times a week now. I'm really at a loss for how to fix this, or why it comes and goes, and I don't want to bug piror with a ticket on it, since as the amateur here, I'm strongly inclined to think the problem is in my code not his.

If anyone knows an experienced webextensions developer with some spare cycles, feel free to point them at this issue.

**Comment by rostwald** [comment]
This bug has become extremely severe lately up to a point where it became near-useless on some profiles.

It's completely random at what combination of "Disable scrolling to change tab" and the TST API-Permission it will work. Sometimes TST-wheel needs to be disabled to work, sometimes the API permission needs to be disabled and most of the times it is endless try & error until it will work again.

I have one firefox profile where it will only work only once a week if i'm very lucky. On my main profile I have to fiddle around with the options 1-2x a day, same on my computer at work where I usually have to restart firefox to get tab scroll working again.

I have the feeling it got much worse after Firefox updating to ~78.6, but whit all the new breakage they introduce with each minor version even on the LTS-branch it's hard to tell exactly...

**Comment by joshuacant** [comment]
I wish I could be of some help here. I actually haven't had this issue happen to me for a long time. I'm running the latest version of TST and I'm on the Dev Edition branch of FF, so I'm at version 87 beta 9. 

I'll put out a general request again: If anyone knows someone with more experience developing and debugging webextensions and thinks they could help, call in a favor and ask them to take a look. Please and thank you.

**Comment by DuBistKomisch** [comment]
FWIW I also haven't seen this issue for a long time, and for me toggling this addon off and on always fixed it. Perhaps you're seeing a different issue?

**Comment by joshuacant** [comment]
I'm closing this issue. It's been a really, really long time since I saw it fail to load or register, and I have to assume the issue was somewhere upstream of my code. Let us all cross our fingers and hope it never comes back. Thanks to everyone who commented with feedback or suggestions, and of course, if it does come back, you can open a new issue on it.
